app/views/homepage/_homepage_header.html.erb

```diff
@@ -0,0 +1,49 @@
+<%
+  add_view_stylesheet("homepage_header")
+%>
+
+<header class="app-c-homepage-header">
```

Comment:
We would also need to update this template to remove the `app-c-` prefixes in the class names